Assets/Scripts/UI/EMCli.cs

```diff
@@ -0,0 +1,184 @@
+﻿using UnityEngine;
+using System.Collections;
+using System;
+using Assets.Scripts.UI;
+using UnityEngine.SceneManagement;
+
+public class EMCli : MonoBehaviour
+{
+    private static Rect windowRect = new Rect(Screen.width - 320f, 0, 300f, 430f);
+    private static GUILayoutOption[] options1 = new GUILayoutOption[] { GUILayout.Width(320f), GUILayout.MaxHeight(430f) };
```

Comment:
Rename property to options. There's just options, so why should we name it options1?